src/gConsent/manage/approveAccessRequest.ts

```diff
@@ -166,6 +167,9 @@ export async function approveAccessRequest(
   options: AccessBaseOptions & {
     returnLegacyJsonld: false;
     verifyLinkedRequest?: boolean;
+    templateMapper?: (
+      templates: string[],
+    ) => UrlString[] | Promise<UrlString[]>;
```

Comment:
Is it necessary to support a potentially async mapper? I would be inclined to keep things simple and expect the mapper to be synchronous for now.